src/test/mochitest/examples/sourcemaps2/main.js

```diff
@@ -1,23 +1,15 @@
-var message = document.getElementById("message");
+var message = document.getElementById('message');
```

Comment:
this a test file, so i'd prefer not change it until we know the tests are passing